test-cases/l2_program_tests/program_and_services.js

```diff
@@ -71,47 +72,29 @@ describe('[Level 2] ServicesAndProgramExistanceTests', () => {
         }
       )
     })
-    it('Should return true if valid openssl version is present', function (done) {
-      if (global.config.edge_build_type == 'snap') {
-        this.skip()
-      }
-      exec(
-        "openssl version | cut -d' ' -f2 | sed 's/[a-z]$//'",
-        (error, stdout) => {
-          if (error) {
-            done(error)
-          } else {
-            assert.equal(
-              compareVersions.compare(
-                stdout.trim(),
-                global.config.program_version.openssl,
-                '>='
-              ),
-              true,
-              'SSL version not valid'
-            )
-            done()
+    if (global.config.edge_build_type != 'snap') {
+      it('Should return true if valid openssl version is present', function (done) {
+        exec(
+          "openssl version | cut -d' ' -f2 | sed 's/[a-z]$//'",
+          (error, stdout) => {
+            if (error) {
+              done(error)
+            } else {
+              assert.equal(
+                compareVersions.compare(
+                  stdout.trim(),
+                  global.config.program_version.openssl,
+                  '>='
+                ),
+                true,
+                'SSL version not valid'
+              )
+              done()
+            }
           }
-        }
-      )
-    })
-    it('Should return true if valid openssh version is present', function (done) {
```

Comment:
why is the open ssh part was removed?

Reply:
We don't need openssh at all for anythig - the version of it really depends fully on what OS version you happen to use.